### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alexwforsythe**
Add installation instructions for Package Control users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ cd path/to/sublime-text/Packages
...
 ```
 
+### [Package Control](https://packagecontrol.io/) users
+
+1. Open command palette > Preferences: Package Control Settings
+2. Add the following to `Package Control.sublime-settings` and save:
+
+```json
+"repositories": [
+	"https://github.com/tinted-theming/tinted-sublime-text"
```
Comment (JamyGolden) [inline README.md]:
```suggestion
    "https://github.com/tinted-theming/tinted-sublime-text"
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,23 @@ cd path/to/sublime-text/Packages
...
+],
+"package_name_map": {
+   "tinted-sublime-text": "tinted_theming",
```
Comment (JamyGolden) [inline README.md]:
```suggestion
    "tinted-sublime-text": "tinted_theming",
```

**Review by JamyGolden — CHANGES_REQUESTED** [review]
Thanks for the PR! Once you accept the spacing adjustments I'll merge

File: README.md (hunk 1 above)
Comment (sergiorussia) [reply README.md]: @JamyGolden maybe it's simpler to accept pr as is and then fix the spacing by yourself? it's been almost 3 weeks and @alexwforsythe does not react, but this is a useful pr for users

Comment (JamyGolden) [reply README.md]: Fair enough. I just commented and forgot about the PR. I’ll merge now.

**Pull request merged**